src/webportal/src/app/job/job-view/fabric/task-attempt.jsx

```diff
@@ -0,0 +1,153 @@
+  const [taskStatus, setTaskStatus] = useState(null);
+
+  const getTimeDuration = (startMs, endMs) => {
+    const start = startMs && DateTime.fromMillis(startMs);
+    const end = endMs && DateTime.fromMillis(endMs);
+    if (start) {
+      return Interval.fromDateTimes(start, end || DateTime.utc()).toDuration([
+        'days',
+        'hours',
+        'minutes',
+        'seconds',
+      ]);
+    } else {
+      return null;
+    }
+  };
+
+  useEffect(() => {
+    fetchTaskStatus(
+      userName,
+      jobName,
+      jobAttemptIndex,
+      taskRoleName,
+      taskIndex,
+    ).then(data => {
+      setTaskStatus(data);
+      setLoading(false);
+    });
+  }, []);
+
+  return (
+    <div>
+      {loading && <SpinnerLoading />}
+      {!loading && (
+        <Stack styles={{ root: { margin: '30px' } }} gap='l1'>
+          <div>
+            <ActionButton
+              iconProps={{ iconName: 'revToggleKey' }}
+              href={`job-detail.html?username=${userName}&jobName=${jobName}`}
+            >
+              Go to Job Detail
+            </ActionButton>
+          </div>
+          <Card style={{ padding: 10 }}>
+            <Stack gap='m'>
+              <Stack horizontal gap='s1' padding='s2'>
+                <Text>Job Name:</Text>
+                <Text>{jobName}</Text>
+              </Stack>
+              <HorizontalLine />
+              <Stack horizontal gap='s1' padding='s2'>
+                <Text>Job Attempt Index:</Text>
+                <Text>{jobAttemptIndex}</Text>
+              </Stack>
+              <HorizontalLine />
+              <Stack horizontal gap='l1' padding='s2'>
+                <Stack gap='m'>
+                  <Text>Task Role</Text>
+                  <Text>{taskRoleName}</Text>
+                </Stack>
+                <Stack gap='m'>
+                  <Text>Task Index</Text>
+                  <Text>{taskIndex}</Text>
+                </Stack>
+                <Stack gap='m'>
+                  <Text>Task Uid</Text>
+                  <Text>{taskStatus.taskUid}</Text>
+                </Stack>
+                <Stack gap='m'>
+                  <Text>Task State</Text>
+                  <StatusBadge status={capitalize(taskStatus.taskState)} />
+                </Stack>
+                <Stack gap='m'>
+                  <Text>Task Retries</Text>
+                  <Text>{taskStatus.retries}</Text>
+                </Stack>
+                <Stack gap='m'>
+                  <Text>Task Created Time</Text>
+                  <Text>
+                    {isNil(taskStatus.createdTime)
+                      ? 'N/A'
+                      : DateTime.fromMillis(
+                          taskStatus.createdTime,
+                        ).toLocaleString(DateTime.DATETIME_MED_WITH_SECONDS)}
+                  </Text>
+                </Stack>
+                <Stack gap='m'>
+                  <Text>Task Duration</Text>
+                  <Text>
```

Comment:
Similar to job detail page, add Task Running Start Time and Task Running Duration?

Reply:
done

Reply:
Just **Add**, and keep existing such as Task Duration, Task Created Time?

---
In reply to: [508961967](https://github.com/microsoft/pai/pull/4966#discussion_r508961967) [](ancestors = 508961967)

Reply:
done